UnitTests/UnitTestController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using GetandTake.Controller;
+using GetandTake.Models;
+using GetandTake.Models.DTOs.BaseDTO;
+using GetandTake.Models.DTOs.ListDTO;
+using GetandTake.Services.Abstract;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnitTests;
```

Comment:
Please, add empty row after this namespace to avoid it overlapping with class:
```suggestion
namespace UnitTests;

public class UnitTestController {
```